packages/suite/src/components/wallet/CoinjoinSummary/BalanceSection.tsx

```diff
@@ -2,12 +2,12 @@ import React from 'react';
 import styled from 'styled-components';
 
 import * as routerActions from '@suite-actions/routerActions';
-import { Account } from '@suite-common/wallet-types';
+import { Account, CoinjoinSession } from '@suite-common/wallet-types';
 import { Card, Translation } from '@suite-components';
 import { useActions, useSelector } from '@suite-hooks';
 import { Button } from '@trezor/components';
-import { CoinjoinSessionStatus } from './CoinjoinSessionStatus';
```

Comment:
You could also delete the file, right? It is replaced by CoinjoinStatus...

Reply:
https://github.com/trezor/trezor-suite/pull/6409/commits/9911012d094a8b339d2a4eda849e085d80041d6f